### PR TITLE
More useful ostree fsck

### PR DESCRIPTION
**Pull request by alexlarsson**
This adds a --all option to not stop on the first error, and prints out what commits each corrupted object is in.

### CODE REVIEW

File: src/ostree/ot-builtin-fsck.c [hunk 1]
```diff
@@ -94,7 +96,12 @@ fsck_one_object (OstreeRepo            *repo,
               (void) ostree_repo_delete_object (repo, objtype, checksum, cancellable, NULL);
               object_missing = TRUE;
             }
-          else
+          else if (opt_all)
+            {
+              *out_found_corruption = TRUE;
+              g_printerr ("%s\n", temp_error->message);
+            }
+          else if (!opt_all)
```
Comment (jlebon) [inline src/ostree/ot-builtin-fsck.c]: Minor: let's just use `else` here? It's clearly equivalent, but it parses better. :)

File: src/ostree/ot-builtin-fsck.c [hunk 2]
```diff
@@ -31,6 +31,7 @@
```
Comment (jlebon) [inline src/ostree/ot-builtin-fsck.c]: This commit is also updating the libglnx submodule to an older commit.

File: src/ostree/ot-builtin-fsck.c [hunk 3]
```diff
@@ -63,16 +63,31 @@ fsck_one_object (OstreeRepo            *repo,
   if (!ostree_repo_fsck_object (repo, objtype, checksum, cancellable, &temp_error))
```
Comment (jlebon) [inline src/ostree/ot-builtin-fsck.c]: Minor: `s/Inclue/Include/` in the commit message.

**Comment by cgwalters** [comment]
I rebased this, fixed the issues jlebon found.  Only thing I'd like to see on top is a basic test for `fsck --all`.

**Comment by cgwalters** [comment]
I added a test case for `--all`.

@rh-atomic-bot r+ a7dd4a6

**Comment by jlebon** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**